pkg/encoding/v1/encoding_test.go

```diff
@@ -1005,21 +1138,6 @@ services:
 		{
 			false, `
 version: 0.1-dev
-services:
```

Comment:
This says that the test should fail if `value` is not provided for the env variable, but this has changed since the value is now a pointer, and the test passed now.
This check is now covered in validations in object.go which checks that exactly one of `value` or `secretRef` has to be provided.